Hospital.Infrastructure.Nats/Extensions/HospitalNatsExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+
+namespace Hospital.Infrastructure.Nats;
```

Comment:
Неймспейс не соответствует пути.
```suggestion
namespace Hospital.Infrastructure.Nats.Extensions;
```